.github/actions/gradle/README.md

```diff
@@ -43,6 +43,29 @@ steps:
 Once the workflow has been triggered and finishes executing, you can navigate to the workflow's output and investigate
 the summary produced by the build validation scripts.
 
+#### Usage with a private repository
+
+The first step of each experiment is to clone the repository containing the build to validate.
+If your repository requires authentication to clone, such as a private repository, you can use the [`actions/checkout`](https://github.com/marketplace/actions/checkout) action to perform the clone instead, as it supports cloning from repositories requiring authentication.
+You can then configure value of the `gitRepo` input parameter to the directory containing the local checkout.
```

Comment:
Should be `...configure the value...` (same for Maven below)

Reply:
Good catch. I've addressed this in `main` with commit https://github.com/gradle/gradle-enterprise-build-validation-scripts/commit/1535fa2f6ace67e50a9bf9caf19284160f3b612b.